app/controllers/account/tasks_controller.rb

```diff
@@ -101,8 +101,12 @@ def complete
     @project = parent
     @task = resource
     @task.update(complete: true)
-    
     respond_to :js
+    @task.watchers.each do |watcher|
+      if watcher != current_user
+        TasksMailer.task_completed(watcher, @task, current_user).deliver
```

Comment:
1. use `after_action` for emails
2. pass task to a mailer and send 1 email for multiple recipients or as CC
3. use `deliver_later` to trigger async job